Logger/Person.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Logger;
+
+public record class Person : BaseEntity
+{
+    //Implemented Name implcitily since FullName is never saved as a property itself so it cant be accessed, so no conflicts
+    public override string Name { get; set; }
+
+    public Person(FullName name)
+    {
+ArgumentException.ThrowIfNullOrWhiteSpace(name);        Name = name.ToString();
```

Comment:
```suggestion
     Name = name.ToString();
```
The parts in Name will actually be check within FullName indiviually so this shouldnt be needed. 
Plus the name passed in through the constructor is not just a string it is a FullName struct so it will break with a this Exception check.